UKHomeOffice/nmsw-ui
# Test UPT external

Added convert excel to json package to read excel file 
Test NMSW-1061 View Help with ETA info on UPT status page
Test NMSW-1060 Search person name
Test to show list of passengers (without UPT status)
## To test
To download convert excel to json package
```
npm ci
```
To run 

```
npm run cypress:run:dev -- --spec cypress/e2e/features/passenger-status-check.feature
```
or
```
npm run cypress:open:dev
```
and select passengers-status-check.feature
and any other tests should work as expected
